webapi/src/main/scala/org/knora/webapi/slice/ontology/domain/service/OntologyRepo.scala

```diff
@@ -16,6 +17,18 @@ import org.knora.webapi.slice.resourceinfo.domain.InternalIri
 trait OntologyRepo extends Repository[ReadOntologyV2, InternalIri] {
 
   override def findById(id: InternalIri): Task[Option[ReadOntologyV2]]
+
   override def findAll(): Task[List[ReadOntologyV2]]
-  def findClassBy(iri: InternalIri): Task[Option[ReadClassInfoV2]]
+
+  def findClassBy(classIri: InternalIri): Task[Option[ReadClassInfoV2]]
```

Comment:
Not sure but wouldn't it be more common to name these methods starting with "get" instead of "find"?

Reply:
As talked about:

`get…` would also be fine. `find` is a common convention for repository methods. IMHO get suggests more of a guarantee that you really _get_ a value whereas find suggests that the might possibly not a value to be returned.